### PR TITLE
[WIP] New UI

### PR DESCRIPTION
**Pull request by paolodamico**
## Changes

The main goal of this PR is introducing the new [standardized components](https://www.figma.com/file/gQBj9YnNgD8YW4nBwCVLZf/PostHog-App?node-id=30%3A2755) and updating the base design to reflect this. 

**Before**
before

**After**
after

The PR introduces a ton of changes because it also refactors a lot of code and removes legacy stuff too. Among the highlights:
- Main styles file went from 471kb minified to 43kb (**> 10X file size reduction**).
- Fully removed legacy Flaticon & Bootstrap (_Note: Bootstrap is kept on the Django views as this will be refactored into React anyways._)
- Refactors the way style files are used, we now have a single global file for styles (`global.scss`) [`styles.scss` has been deprecated] that apply universally, and every other style is applied in local files that are imported directly by the relevant components (i.e. `styles/` folder disappears). All style configurations are defined on `vars.scss` to be able to import the same variables into other style files.
- Ant is now imported using LESS instead of plain CSS to override styling gracefully.
- Removes custom components that re-implement functionality available on Ant (e.g. Dropdown, Card, ...)
- Refactored persons & cohorts endpoint from `/people/persons` -> `/persons` & `/people/cohorts` to `/cohorts`

When reviewing please be sure to test **everything, everything, everything**. This PR affects almost every single view in the frontend. Also, to avoid bloating the PR even more, some minor UI bugs that were already present in the app were carried through in this PR, please keep in mind if you encounter any minor bug.

## Checklist

- [X] **N/A**. All querysets/queries filter by Organization, Team, and User (if this PR affects ANY querysets/queries).
- [X] **N/A**. Django backend tests (if this PR affects the backend).
- [X] Cypress end-to-end tests (if this PR affects the frontend).

### CODE REVIEW

File: frontend/src/scenes/events/EventsTable.js
Comment (timgl) [inline frontend/src/scenes/events/EventsTable.js:189]: Not sure about this <Card> around the table, takes up a lot of space and looks a bit odd I think?
image

Comment (timgl) [reply frontend/src/scenes/events/EventsTable.js:189]: Ah looks like it's on some other tables too. I do think it looks good on the setup/settings pages etc, but not sure about the tables.

**Comment by yakkomajuri** [comment]
Thanks a lot for this! 

So a few thoughts from a quick first UX review:

# Actual "Bugs"

## Broken Tooltip

Screenshot 2020-11-03 at 11 29 35

Screenshot 2020-11-03 at 11 29 42

## Trash Icon Color (/actions)

Screenshot 2020-11-03 at 11 31 54

# Personal Comments

## Topbar content too small
Screenshot 2020-11-03 at 11 30 34

## Color on the tables is a little too playful

Screenshot 2020-11-03 at 11 30 21

## Unsure how to feel about the table borders

Screenshot 2020-11-03 at 11 31 14

**Review by timgl — COMMENTED** [review]
I think I prefer having a bit of a grey bar here to give a bit of contrast between controls and the graph itself:
image

Spacing here is a bit odd:
image

This happened after a toast appeared
image


Maybe funnels are broken? Seems like it gets stuck 'loading' despite the network request finishing

There's an odd line here below the URL. I also think with less contrast this screen looks a bit more cluttered
image

File: frontend/src/scenes/events/EventsTable.js
Comment (yakkomajuri) [reply frontend/src/scenes/events/EventsTable.js:189]: Agree with @timgl on the above

**Comment by Twixes** [comment]
Really nice! A few notes:

1. The logo in top-left is misaligned because none of its classes `row logo-row d-flex align-items-center justify-content-center` are styled now. Should be enough to just bring this back to the logo `div`:
    ```CSS
    display: flex;
    justify-content: center;
    align-items: center;
    ```
    Misaligned logo

1. These buttons could maybe be a bit bigger?
     Buttons

1. The command palette and the sidebar used to have the same bluish gray background, but now the palette has neutral gray, which looks too warm and out of place.
     Sidebar vs. Palette

1. Sessions, org invites and instance licenses don't follow the new bordered style of pages.

**Comment by lottiecoxon** [comment]
Would it be worth both 'add filter' buttons to be rounded? Also some spacing between the mini add filters button and the 'add graph series' would be good - feels a little squished at the moment 


Screenshot 2020-11-03 at 12 13 46

**Comment by lottiecoxon** [comment]
Please can we put the title text  and paragraph text starting on the same line - it looks strange that they are on different levels + there is a very sad 'orphan' word hanging off the edge by itself (not good to have ). To fix this I would change the ampersand to the word 'and' and move 'standardised' down another level?

Screenshot 2020-11-03 at 12 47 04

**Comment by lottiecoxon** [comment]
This could do with some spacing between the save button and the text below 

Screenshot 2020-11-03 at 12 50 44

**Comment by lottiecoxon** [comment]
Screenshot 2020-11-03 at 12 53 30
I would personally bring the opacity up a little bit- as this seems on the edge of not being seen?

**Comment by jamesefhawkins** [comment]
# Spacing

Before

Screenshot 2020-11-03 at 15 42 43

After

Screenshot 2020-11-03 at 15 42 35

We've increased the spacing to the page title here, but it's pushing the useful content further out the bottom of the page. Is this move intentional?

# Table borders

I'd agree with @timgl above - this extra border doesn't add anything functionality-wise thus adds clutter. Not sure if deliberate. 
Screenshot 2020-11-03 at 15 47 20

# 'Add' buttons shifted to right hand side

Just a little note to say I'm a fan of this

before
Screenshot 2020-11-03 at 15 53 20

after

Screenshot 2020-11-03 at 15 53 42

# toasts

This font feels a little out of place - it's weirdly bold I think?

Screenshot 2020-11-03 at 15 56 26

# settings 

This isn't a new issue, ut the spacing between this 'Save' button and the header 'Data Capture Configuration' looks too little.

Should we have an HR here like we do between PostHog Toolbar and ie Session Recording?

Screenshot 2020-11-03 at 15 57 22

**Comment by jamesefhawkins** [comment]
Screenshot 2020-11-03 at 15 59 07

Also, font weights here - ie PostHog Toolbar is heavier weight than Session Recording, which feels wrong.

**Comment by paolodamico** [comment]
**Thank you all for the feedback, super helpful 🚀 !**

All the feedback has been addressed and this is ready for another pass. I'll resolve the merge conflicts too and fix the test that's not passing.

Two small things I'd like to push back a bit on.

> Color on the tables is a little too playful
from @yakkomajuri,
- As these are links we use the same global color to show they can be clicked. I'm a bit wary of using another color to indicate text is clickable to avoid the same jumble of styles we had before.

> I would personally bring the opacity up a little bit- as this seems on the edge of not being seen?
from @lottiecoxon,
- This has not been changed and uses the muted text color we designated (#d9d9d9), happy to revisit, but might be worth updating the text muted color everywhere. Thoughts?

**Review by Twixes — APPROVED** [review]
Looks all good to me!

**Comment by paolodamico** [comment]
Thanks @Twixes, I'll wait until the stand up to merge in case someone else has any additional feedback as this is quite a large PR.

**Comment by timgl** [comment]
I'm in favour of merging now, the longer this is open the more merge conflicts there'll be. Can always open another one

**Pull request merged**
